Backpacks.cs

```diff
@@ -4732,14 +4734,14 @@ public ItemContainerAdapter()
                 };
             }
 
-            public ItemContainerAdapter Setup(Backpack backpack, int pageIndex, ItemContainer container)
+            public ItemContainerAdapter Setup(Backpack backpack, int pageIndex, StorageContainer container)
```

Comment:
Rename the `container` variable to `storageContainer`, since `container` is typically used for just `ItemContainer` instances throughout the plugin. Having consistent naming for variables of a given type helps with readability.